libmamba/src/core/shell_init.cpp

```diff
@@ -760,6 +841,18 @@ namespace mamba
             std::ofstream sh_file = open_ofstream(sh_source_path);
             sh_file << data_mamba_fish;
         }
+        else if (shell == "nu")
+        {
+            NuActivator a{ context };
+            auto sh_source_path = a.hook_source_path();
+            try
+            {
+                fs::create_directories(sh_source_path.parent_path());
+            }
+            catch (...)
+            {
+            }
```

Comment:
Maybe log, or let the exception break the current flow?

Reply:
I just pushed a comment to this part. Does this suffice?